src/blueprints/main.py

```
@@ -323,7 +317,7 @@ def display_next():
             except Exception:
                 pass
     except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)